### DISCUSSION TITLE
Using RFID MFRC522 on SPI1_3CS

### DISCUSSION THREAD

**Discussion by benni26** [MiczFlor/RPi-Jukebox-RFID#2156] [Q&A]
Hi,

I am trying to get the RFID-reader RC522 working on SPI1. By default SPI0 is used, but since I am using this display https://www.waveshare.com/wiki/3.5inch_RPi_LCD_(A) the default SPI port is allready used. 
I added the following line in the boot/config.txt
`dtoverlay=spi1-3cs`

and `ls /dev/spi*` shows
`/dev/spidev1.0  /dev/spidev1.1  /dev/spidev1.2`

But I cant get it working. Do you have any Idea?

**Comment by s-martin** [comment]
Did you check https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Wiring_for_RC522_card_reader , if this helps?
